### PR TITLE
Export 2017-07-28

### PR DESCRIPTION
**Pull request by pocmo**
Export of all strings planned for the next release.

### CODE REVIEW

File: locales/templates/app.pot [hunk 1]
```diff
@@ -219,11 +231,8 @@ msgstr ""
 #. content to show up in the "recent apps" screen      and prevents the user
 #. from taking screenshots or casting the app to other displays. %1$s will be
 #. replaced with the      name of the app (e.g. Firefox Focus).
-#, c-format
 msgctxt "preference_privacy_secure_mode_summary"
-msgid ""
-"Turn off stealth mode to take screenshots and see sites you’ve visited "
-"using %1$s in your Recent Apps list"
+msgid "Hide webpages when switching apps"
```
Comment (flodolo) [inline locales/templates/app.pot:235]: This doesn't look right. The string completely changed, but the ID and comment remained the same?

The comment is completely off at this point.

Comment (pocmo) [reply locales/templates/app.pot:235]: It was decided to change the description completely (The setting is still the same). User testing revealed that the previous explanation was too complex and users didn't understand it. So the UX team decided to replace it with a new simpler explanation. Should we create a new key for it and remove the old one?

Comment (flodolo) [reply locales/templates/app.pot:235]: I think it would be more transparent. Also need to update that description.

Comment (pocmo) [reply locales/templates/app.pot:235]: Okay, new PR incoming!

**Pull request closed without merging**